geode-core/src/main/java/org/apache/geode/internal/cache/LocalRegionDataView.java

```diff
@@ -381,10 +381,7 @@ public void postRemoveAll(DistributedRemoveAllOperation op, VersionedObjectList
     // BR, DR's removeAll
     long token = -1;
     try {
-      if (reg.getServerProxy() != null || (reg.getDataPolicy() != DataPolicy.NORMAL
-          && reg.getDataPolicy() != DataPolicy.PRELOADED)) {
-        token = reg.postRemoveAllSend(op, successfulOps);
-      }
+      token = reg.postRemoveAllSend(op, successfulOps);
```

Comment:
Should not remove the reg.getServerProxy() != null check here.

Reply:
I changed it back to the code before I fix 7702 for this part.